app/src/main/res/values/colors.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
     <color name="colorPrimary">#47ECAA</color>
     <color name="colorPrimaryDark">#47ECAA</color>
-    <color name="colorAccent">#D81B60</color>
+    <color name="colorAccent">@color/colorPrimary</color>
```

Comment:
C'est un peu violent ça, ça va changer tout le look de l'app ?

Reply:
c'est juste que colorAccent " used more subtly throughout the app, to call attention to key elements" etait rouge or cela ne correspond pas vraiment avec les couleurs que l'on utilise pour l'instant. Du coup pour l'instant je lui assigne la valeur de la couleur primaire

Reply:
il y a beaucoup de pour l'instant..